test/html/vod_miniplayer.tpl.html

```diff
@@ -101,14 +101,34 @@
             } : {})
         });
 
+        console.log('%cTEST INSTANCE', 'color: #fff; font-weight: bold; background-color: #BADA55; padding: 3px 6px; border-radius: 3px;', instance);
+
         instance.on('miniPlayerToggle', (event) => console.log(`[Event API] miniPlayerToggle`, event, event.detail.isToggledOn));
 
         Array.from(document.getElementById('setupForm').children).forEach(child => child.remove());
         const refreshPage = document.createElement('button');
         refreshPage.onclick = () => location.reload();
-        refreshPage.innerText = 'Reset';
+        refreshPage.innerText = 'Reset Test';
```

Comment:
Should this still have "test" in it?

Reply:
Yeah, it's the test file, the button is to reload the page to start the test from the start :)

Reply:
In that case, approved. LGTM